File: 문자열/백준_1427_소트인사이드/소트인사이드.java
import java.util.*;

public class Main{

    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int[] arr = new int[10];
        Arrays.fill(arr, 10);
        int i = 0;

        while(n >= 1){
            arr[i] = n % 10;
            i++;
            n /= 10;
        }
        Arrays.sort(arr);
        for(int j = i-1; j >= 0; j--){
            System.out.print(arr[j]);
        }

    }
}

File: 문자열/백준_1427_소트인사이드/소트인사이드.kt
import java.util.*

fun main(args: Array<String>) {
    var s = Scanner(System.`in`)
    var n = s.nextInt()
    var arr = IntArray(10, {0})
    var i = 0

    while(n >= 1){
        arr[i] = n% 10
        i+=1
        n /= 10
    }
    arr.sortDescending()
    for (j in 0 until i) print(arr[j])

}